napari/utils/colormaps/_tests/test_colormap.py

```diff
@@ -101,3 +101,11 @@ def test_colormap_equality():
 def test_colormap_recreate():
     c_map = Colormap("black")
     Colormap(**c_map.dict())
+
+
+@pytest.mark.parametrize('ndim', range(1, 5))
+def test_mapped_shape(ndim):
+    img = np.random.random((5,) * ndim)
```

Comment:
Sorry, I missed this the first time, but prefer not to use unseeded randoms to keep tests deterministic.

```suggestion
    np.random.seed(0)
    img = np.random.random((5,) * ndim)
```

Alternatively, `np.ones((5,) * ndim)` (or `zeros`) could be fine and faster here.

Reply:
I think this should be resolved before the merge.